docs/ocis/release_notes.md

```diff
@@ -7,6 +7,29 @@ geekdocEditPath: edit/master/docs/ocis
 geekdocFilePath: release_notes.md
 ---
 
+## ownCloud Infinite Scale 1.4.0 Technology Preview
+
+Version 1.4.0 brings new features, bug fixes and further improvements. The accessibility of ownCloud Web has greatly improved, paving the way for WCAG 2.1 compliance. The Infinite Scale platform has received major improvements regarding memory consumption. The user storage quota feature has been implemented and folder sizes are now properly calculated. It is now possible to write log messages to log files and to specify configuration values using a config file.
+
+The most prominent changes in version 1.4.0 comprise:
+
+- ownCloud Web has received many further improvements regarding accessibility and WCAG 2.1 compliance
+- Reduced the memory consumption of the runtime drastically (by a factor of 24) [#1762](https://github.com/owncloud/ocis/pull/1762)
+- Initial quota support to impose storage space restrictions for users (query / set) [#1405](https://github.com/cs3org/reva/pull/1405)
+- Folder sizes are now calculated correctly (tree size accounting) [#1405](https://github.com/cs3org/reva/pull/1405)
+- The media viewer now also works with upper case file extensions (e.g., JPG) [#4627](https://github.com/owncloud/web/pull/4627)
```

Comment:
That's not included in the ocis release!

Reply:
fixed